keyvalues/keyvalues.go

```diff
@@ -39,3 +40,14 @@ func Parse(src []string, allowEmptyValues bool) (map[string]string, error) {
 	}
 	return results, nil
 }
+
+// Join returns a sorted slice containing each item in the given map, as
+// "key=value" strings.
+func Join(m map[string]string) []string {
```

Comment:
Is there a reason to sort?

Reply:
Repeatable output, since map ordering is not guaranteed.